src/Infrastructure/Controller/ErrorController.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare(strict_types=1);
+
+namespace StepupReadId\Infrastructure\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Response;
+use Throwable;
+
+class ErrorController extends AbstractController
+{
+    public function __invoke(Throwable $exception): Response
+    {
+        return $this->render('default/error.html.twig', [
+            'message' => $exception->getMessage(),
```

Comment:
Is this desirable? Showing the exception message without being certain what text may be on there? We may run a risk at disclosing sensitive data here.

Reply:
Changed it to use predefined messages